From 4dee9abe1d000d0fc266924949ea7b97e025d7f7 Mon Sep 17 00:00:00 2001
From: Dan Williams <dcbw@redhat.com>
Date: Fri, 7 Aug 2009 10:01:50 -0500
Subject: [PATCH 1/3] bluetooth: fix device capability checking (need NAP not PANU)

The remote device (which is what we're checking) needs to provide
the NAP service (meaning it can bridge packets to the cellular
network), not the PANU service.
---
 src/gnome-bluetooth/network-manager-applet.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/gnome-bluetooth/network-manager-applet.c b/src/gnome-bluetooth/network-manager-applet.c
index 3cdd5b5..58f5b9c 100644
--- a/src/gnome-bluetooth/network-manager-applet.c
+++ b/src/gnome-bluetooth/network-manager-applet.c
@@ -45,7 +45,7 @@ has_config_widget (const char *bdaddr, const char **uuids)
 
 	for (i = 0; uuids && uuids[i] != NULL; i++) {
 		g_message ("has_config_widget %s %s", bdaddr, uuids[i]);
-		if (g_str_equal (uuids[i], "PANU"))
+		if (g_str_equal (uuids[i], "NAP"))
 			return TRUE;
 	}
 	return FALSE;
-- 
1.6.2.5

